Workbooks/AzureActivity.json

```diff
@@ -87,7 +87,7 @@
               "selectAllValue": "All"
             },
             "queryType": 0,
-            "resourceType": "microsoft.resources/resourcegroups"
+            "resourceType": "microsoft.operationalinsights/workspaces"

```

Comment:
You should update the version of the workbook in the "workbooksMetadata" file, so that customers that have already saved this workbook will be able to update to the new version.

Reply:
Done, please let me know for any more modifications.